core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -533,21 +533,15 @@ private[log] class LogCleanerManager(val logDirs: Seq[File],
   def maintainUncleanablePartitions(): Unit = {
```

Comment:
ditto. `LogCleanerManagerTest` is a good place :)

Reply:
`maintainUncleanablePartitions` is invoked by `CleanerThread#doWork`, which is a daemon thread. It has been ㄎwidely tested through integration tests and unit tests related to LogCleaner. IMHO, we can leverage these existing tests and don't need to write new ones.